src/zshrc

```diff
@@ -170,6 +170,15 @@ bindkey -M menuselect '^o' accept-and-infer-next-history
 bindkey '^[[A' history-substring-search-up
 bindkey '^[[B' history-substring-search-down
 
+# Use the provided Bash autocompletion for adr-tools. Note that there are other
+# scripts in this directory, but not all of them work well with zsh.
+autoload -U +X bashcompinit && bashcompinit
+source /opt/homebrew/etc/bash_completion.d/adr-tools
```

Comment:
There is only [one autocomplete script](https://github.com/npryce/adr-tools/tree/master/autocomplete) packaged in `adr-tools` and it didn't work for me out of the box, so I assume so.

Reply:
I think that makes sense. I'm kind of loathe to introduce special casing. I wonder if this is an upstream issue actually? Don't worry too much though if this is a yak not worth shaving.

https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/adr-tools.rb#L25

https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/git.rb#L132

Reply:
OK, I'll have a go at an upstream PR, and if that gets merged, I'll close this one.

Reply:
So, this has been a little more complicated than I'd expected. 

I have a completions script that works for `zsh`, but [`adr-tools`](https://github.com/npryce/adr-tools) last had a commit pushed to its default branch in 2020, and it currently has 25 open PRs, and 502 forks. I don't think there's any real chance of getting a change merged in.

Homebrew (quite sensibly) don't accept patches that have not been merged into upstreams. 

We could start fork number 503, add the patch, and try to switch the packages in Homebrew, but I'm not sure it's really worth it. 

So, I think the choices here are:

1. Add the Bash completions script, as per this PR.
2. Add a comment to the `.zshrc` file or some documentation somewhere on how to fix this, for the few people who might need it.
3. Just close this PR without action, and leave the next person to be equally confused.

None of these feels especially attractive TBH.

Reply:
I think option 1 feels the best, though suggest we use `$(brew --prefix)` instead of hardcoding the path (this won't work on an Intel computer as it is).